src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -215,5 +229,41 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
             Assert.Equal(0, target.ExitCode);
             Assert.Equal(0, result);
         }
+
+        [Fact]
+        public void StrykerCLI_WithNoFilesToExcludeSet_ShouldPassDefaultValueToStryker()
+        {
+            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
+            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>()));
+
+            var target = new StrykerCLI(mock.Object);
+
+            target.Run(new string[] { });
+
+            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
+                !o.FilesToExclude.Any())));
+        }
+
+        [Theory]
+        [InlineData("--files-to-exclude")]
+        [InlineData("-fte")]
+        public void StrykerCLI_WithFilesToExcludeSet_ShouldPassFilesToExcludeToStryker(string argName)
+        {
+            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
+            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>()));
+
+            var target = new StrykerCLI(mock.Object);
+
+            target.Run(new[] { argName, "['.\\StartUp.cs','./ExampleDirectory/Recursive.cs', '.\\ExampleDirectory/Recursive2.cs']" });
+
+            var firstFileToExclude = Path.Combine(_currentDirectory, "StartUp.cs");
+            var secondFileToExclude = Path.Combine(_currentDirectory, "ExampleDirectory", "Recursive.cs");
+            var thirdFileToExclude = Path.Combine(_currentDirectory, "ExampleDirectory", "Recursive2.cs");
+
+            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o =>
```

Comment:
You could use shouldly to verify the array nicer.

Reply:
Oh, nice! Didn't know of that one :-)!